benchmark failed because of the update of protobuf's version

See the detailed error:
```
Linking CXX executable inference
/home/work/.jumbo/bin/cmake -E cmake_link_script CMakeFiles/inference.dir/link.txt --verbose=1
/home/work/liuyiqun/install/android/toolchains/arm64-android-21/bin/aarch64-linux-android-g++  --sysroot=/home/work/liuyiqun/install/android/toolchains/arm64-android-21/sysroot  -ffunction-sections -fdata-sections -march=armv8-a  -O3 -DNDEBUG  -pie -fPIE -Wl,--gc-sections  CMakeFiles/inference.dir/inference.cc.o  -o inference  -L/home/work/liuyiqun/PaddlePaddle/Paddle/build_paddle/dist_android/lib/arm64-v8a  -L/home/work/liuyiqun/PaddlePaddle/Paddle/build_paddle/dist_android/third_party/gflags/lib/arm64-v8a  -L/home/work/liuyiqun/PaddlePaddle/Paddle/build_paddle/dist_android/third_party/glog/lib/arm64-v8a  -L/home/work/liuyiqun/PaddlePaddle/Paddle/build_paddle/dist_android/third_party/protobuf/lib/arm64-v8a  -L/home/work/liuyiqun/PaddlePaddle/Paddle/build_paddle/dist_android/third_party/zip/lib/arm64-v8a  -L/home/work/liuyiqun/PaddlePaddle/Paddle/build_paddle/dist_android/third_party/openblas/lib/arm64-v8a  -Wl,--start-group -Wl,--whole-archive -lpaddle_capi_layers -Wl,--no-whole-archive -lpaddle_capi_engine -Wl,--end-group -lglog -lgflags -lprotobuf -lz -lopenblas 
/home/work/liuyiqun/install/android/toolchains/arm64-android-21/bin/../lib/gcc/aarch64-linux-android/4.9.x/../../../../aarch64-linux-android/bin/ld: /lib64/libz.so.1: no version information available (required by /home/work/liuyiqun/install/android/toolchains/arm64-android-21/bin/../lib/gcc/aarch64-linux-android/4.9.x/../../../../aarch64-linux-android/bin/ld)
/home/work/liuyiqun/PaddlePaddle/Paddle/build_paddle/dist_android/third_party/protobuf/lib/arm64-v8a/libprotobuf.a(common.cc.o): In function `google::protobuf::internal::DefaultLogHandler(google::protobuf::LogLevel, char const*, int, std::string const&)':
common.cc:(.text._ZN6google8protobuf8internal17DefaultLogHandlerENS0_8LogLevelEPKciRKSs[_ZN6google8protobuf8internal17DefaultLogHandlerENS0_8LogLevelEPKciRKSs]+0xc0): undefined reference to `__android_log_write'
common.cc:(.text._ZN6google8protobuf8internal17DefaultLogHandlerENS0_8LogLevelEPKciRKSs[_ZN6google8protobuf8internal17DefaultLogHandlerENS0_8LogLevelEPKciRKSs]+0x12c): undefined reference to `__android_log_write'
collect2: error: ld returned 1 exit status
make[2]: *** [inference] Error 1
make[2]: Leaving directory `/home/work/liuyiqun/PaddlePaddle/Mobile/benchmark/tool/C/build_android'
make[1]: *** [CMakeFiles/inference.dir/all] Error 2
make[1]: Leaving directory `/home/work/liuyiqun/PaddlePaddle/Mobile/benchmark/tool/C/build_android'
make: *** [all] Error 2
```

Need to link Android's `log` library.

--- a/benchmark/tool/C/CMakeLists.txt
+++ b/benchmark/tool/C/CMakeLists.txt
@@ -41,7 +41,8 @@ if(ANDROID_ABI)
     target_link_libraries(${PROJECT_NAME}
                           -Wl,--start-group -Wl,--whole-archive -lpaddle_capi_layers
                           -Wl,--no-whole-archive -lpaddle_capi_engine -Wl,--end-group
-                          ${THIRD_PARTY_LIBRARYS})
+                          ${THIRD_PARTY_LIBRARYS}
+                          log)
 else()
     target_link_libraries(${PROJECT_NAME}
                     -Wl,--start-group -Wl,--whole-archive -lpaddle_capi_whole

